### ISSUE TITLE
No high quality, just 128kb

### ISSUE THREAD

**Issue by suntification** [G3VV/Yank#7]
Hi,
Thanks for your effort on this script but it seems the PyDeezer you used in your project doesn't able to provide high quality output for any music.
so any Idea?

Best

**Comment by G3VV** [comment]
I do think you'd need to input a Deezer ARL with premium on the account for higher quality music

**Comment by suntification** [comment]
provided arl is for a premium account

**Comment by G3VV** [comment]
What is the KBPS of the downloaded song?

**Comment by suntification** [comment]
128 kbps, no matter what you pass , I tried my arl with other packages like deemix or streamrip and they work fine.

**Comment by G3VV** [comment]
I'll have a look into this and see what I can do

**Comment by TwentySeven28** [comment]
I am experiencing the same issue.

**Comment by afkarxyz** [comment]
I think this issue is caused by pydeezer, maybe you should try changing another library, because I tried pydeezer with arl premium also the result is 128kbps. https://github.com/acgonzales/pydeezer/issues/30

**Comment by suntification** [comment]
Yes it's because of pydeezer itself, I moved to deemix and everything works smoothly

**Comment by bverkron** [comment]
Not Deezer related but I noticed that Spotify songs are also only ever 128 kbps as far as I can tell, at least via the demo site. Are they higher quality running locally and providing your own developer token

**Comment by afkarxyz** [comment]
> Not Deezer related but I noticed that Spotify songs are also only ever 128 kbps as far as I can tell, at least via the demo site. Are they higher quality running locally and providing your own developer token

Yes, it can reach up to 320 kbps with a premium account. You can use **OnTheSpot** if you want to run it locally: [https://github.com/justin025/onthespot](https://github.com/justin025/onthespot)

**Comment by bverkron** [comment]
That's great, thanks.

**Comment by G3VV** [comment]
> Not Deezer related but I noticed that Spotify songs are also only ever 128 kbps as far as I can tell, at least via the demo site. Are they higher quality running locally and providing your own developer token

Yeah the demo site is only 128kbps, but I know the library to download songs (pydeezer) is an issue and is not maintained anymore. 
I'm planning on either finding a new library to fix this or coding one myself, either will happen eventually.
